meshes/pan_head.dae

```diff
@@ -0,0 +1,191 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Can Blender import a dae? If so, I'm wondering if it would make sense to exclude the .blend files from the repo?